templates/webapps/galaxy/workflow/run.mako

```diff
@@ -576,15 +576,14 @@ if wf_parms:
 
 %if has_upgrade_messages:
 <div class="warningmessage">
-    Problems were encountered when loading this workflow, likely due to tool
-    version changes. Missing parameter values have been replaced with default.
-    Please review the parameter values below.
+    Warning: Some tools in this workflow have been updated. The workflow may still work, but any new inputs will have default values.
```

Comment:
Great suggestions here... I've incorporated most feedback in the latest commit I think.